.github/workflows/contracts.yml

```diff
@@ -21,6 +21,7 @@ env:
 
 jobs:
   contracts:
+    timeout-minutes: 60
```

Comment:
@mrain @alxiong is this a reasonable timeout for this job?

Reply:
cc @alysiahuggins @philippecamacho do you have more inputs?

Reply:
Actually I think this is more than reasonable, anything longer than an hour shouldn't block CI (an hour is arguably pretty generous already).

Reply:
Right, I think it's more than enough

Reply:
Looking at healthy tests I see they pass in ~11min, can we reduce this to ~15 or ~20?

Reply:
Here you can see they consistently succeed in <= 12 min. https://github.com/EspressoSystems/espresso-sequencer/actions/workflows/contracts.yml

Reply:
done https://github.com/EspressoSystems/espresso-sequencer/pull/1216/commits/5ea2b330911d7949b83d7d12bd2f50c9863e1209